eng/Versions.props

```diff
@@ -72,9 +72,12 @@
     <MicrosoftVisualStudioLanguageServicesPackageVersion>4.7.0-2.23301.2</MicrosoftVisualStudioLanguageServicesPackageVersion>
     <MicrosoftSourceLinkGitHubPackageVersion>8.0.0-beta.23252.2</MicrosoftSourceLinkGitHubPackageVersion>
     <MicrosoftDotNetXliffTasksPackageVersion>1.0.0-beta.23266.1</MicrosoftDotNetXliffTasksPackageVersion>
-    <!-- Exception - this version is not updated by automation, but is in Version.Details due to source-build.
-        See its entry in Version.Details for more information. -->
+    <!--
+      Exception - Microsoft.Extensions.ObjectPool and System.Collections.Immutable packages are not updated by automation,
+      but are present in Version.Details.xml for source-build PVP flow. See the comment in Version.Details.xml for more information.
+    -->
     <MicrosoftExtensionsObjectPoolPackageVersion>6.0.0</MicrosoftExtensionsObjectPoolPackageVersion>
+    <SystemCollectionsImmutablePackageVersion>6.0.0</SystemCollectionsImmutablePackageVersion>
```

Comment:
This repo needs to be on SCI 7.0, not 6.0

Reply:
It is currently on 6.0.0 - I'm only moving the property to a different group within the file.

Should update to SCI 7.0.0 be part of a separate PR?

Reply:
I think Jared is asking what will be the impact to source-build of upgrading it?

Reply:
No impact on source-build - it always picks the latest live version anyway - 8.0.x